src/cli.ts

```diff
@@ -51,8 +51,12 @@ const cli = meow(`
         
         repos <subcommand> [--force]
             Handles repo specific actions where <subcommand> is one of the following:
-            --update|-u [--nofetch] [--reset-to-remote]:
-                Updates all parent repos.
+            --update|-u [--nofetch] [--reset-to-remote]: Updates all parent repos. 
+                Tags instead of branches are supported in parent-repos.json by cplace-cli >0.17.0 e.g.:   
```

Comment:
I would just add to the documentation here that using tags causes branches to be created locally which will not be removed automatically.